satellite/console/database.go

```diff
@@ -27,6 +27,12 @@ type DB interface {
 	ResetPasswordTokens() ResetPasswordTokens
 	// UsageRollups is a getter for UsageRollups repository
 	UsageRollups() UsageRollups
+	// UserPaymentInfos is a getter for UserPaymentInfos
+	UserPaymentInfos() UserPaymentInfos
+	// ProjectPaymentInfos is a getter for ProjectPaymentInfos
+	ProjectPaymentInfos() ProjectPaymentInfos
+	// ProjectInvoiceStamps is a getter for ProjectPaymentInfos
```

Comment:
There's a typo here. Should `ProjectInvoiceStamps is a getter for ProjectPaymentInfos` be `ProjectInvoiceStamps is a getter for ProjectInvoiceStamps`?

Reply:
yup, you are right